com.vmware.horizon/com.vmware.horizon.yaml

```diff
@@ -0,0 +1,55 @@
+---
+app-id: com.vmware.horizon
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: vmware-view
+finish-args:
+  - "--share=ipc"
+  - "--device=dri"
+  - "--socket=x11"
+  - "--share=network"
+  - "--env=LD_LIBRARY_PATH=/app/lib:/app/lib/vmware"
+  - "--env=GDK_BACKEND=x11"
+  - "--env=ENABLE_FOLDER_REDIRECTION=true"
+build-options:
+  env:
+    VERSION: 2106-8.3.0-18251983
+modules:
+  - name: glib
+    buildsystem: meson
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/glib/2.69/glib-2.69.1.tar.xz
+        sha256: f92f34057a091fc8638d91f10cece842cb8618e9a1090b0ddb19cc15a21bf39c
+  - name: vmware-horizon-client
+    buildsystem: simple
+    sources:
+      - type: archive
+        url: https://download3.vmware.com/software/view/viewclients/CART22FQ2/VMware-Horizon-Client-Linux-2106-8.3.0-18251983.tar.gz
```

Comment:
Is it actually redistributable?

Reply:
I beleive it's not, though not sure if extra-data is warranted considering we're downloading a tarball. Shall I  swap to that?